buildSrc/src/main/java/Config.kt

```diff
@@ -104,8 +108,11 @@ object Config {
         val graphQlJava = "com.graphql-java:graphql-java:17.3"
 
         val kotlinReflect = "org.jetbrains.kotlin:kotlin-reflect"
+        val kotlinStdLib = "org.jetbrains.kotlin:kotlin-stdlib"
 
         val navigationRuntime = "androidx.navigation:navigation-runtime:2.4.2"
+        // compose deps
+        val composeNavigation = "androidx.navigation:navigation-compose:2.4.2"
```

Comment:
If they follow the same versioning, we can extract `2.4.2` to a const.

Reply:
good call, yeah they do